api/envoy/config/filter/network/tcp_proxy/v2/tcp_proxy.proto

```diff
@@ -165,4 +173,6 @@ message TcpProxy {
   // load balancing algorithms will select a host randomly. Currently the number of hash policies is
   // limited to 1.
   repeated type.HashPolicy hash_policy = 11 [(validate.rules).repeated = {max_items: 1}];
+
+  TunnelingConfig tunneling_config = 12;
```

Comment:
Can you add docs here, mainly saying what the default is if nothing is specified?